theano/tensor/raw_random.py

```diff
@@ -520,13 +520,8 @@ def binomial(random_state, size=None, n=1, p=0.5, ndim=None,
     n = tensor.as_tensor_variable(n)
     p = tensor.as_tensor_variable(p)
     ndim, size, bcast = _infer_ndim_bcast(ndim, size, n, p)
-    if n.dtype == 'int64':
-        # THIS WORKS AROUND A NUMPY BUG on 32bit machine
-        # Erase when the following works on a 32bit machine:
```

Comment:
@nouiz I'm not sure what you mean by that. What should be in a test?
As far as I understand it, the comment is not about a regression, it is about something that we hoped would be changed in the future.